performance/main.py

```diff
@@ -250,6 +250,74 @@ class ArrayGOPerfREF(ArrayGOPerf):
     entry = staticmethod(ArrayGOREF)
 
 
+#-------------------------------------------------------------------------------
+class IsNaElementPerf(Perf):
+    NUMBER = 1000
+
+    def pre(self):
+        class FloatSubclass(float): pass
+
+        self.values = [
+                # Na-elements
+                np.datetime64('NaT'), np.timedelta64('NaT'), None, float('NaN'), -float('NaN'),
+
+                # Non-float, Non-na elements
+                1, 'str', np.datetime64('2020-12-31'), datetime.date(2020, 12, 31), False,
+        ]
+
+        nan = np.nan
+        nanjs = [
```

Comment:
Maybe `complex_nans`? `nanjs` sort of implies that only the complex part is NaN:

```suggestion
        complex_nans = [
```